src/linux/epoll.rs

```diff
@@ -301,6 +301,7 @@ impl Epoll {
         timeout: i32,
         events: &mut [EpollEvent],
     ) -> io::Result<usize> {
```

Comment:
How about returning an error instead of asserting?

Reply:
resolved by removing parameter